standalone/src/init.ts

```diff
@@ -106,12 +106,16 @@ function connectCall(channelID: string, callTitle: string, wsURL: string, iceCon
         window.callsClient.init(channelID, callTitle).then(() => {
             window.callsClient.ws.on('event', wsEventHandler);
         }).catch((err: Error) => {
-            delete window.callsClient;
             logErr(err);
+            if (closeCb) {
+                closeCb();
+            }
         });
     } catch (err) {
-        delete window.callsClient;
         logErr(err);
+        if (closeCb) {
+            closeCb();
+        }
```

Comment:
Unrelated to the PR but fixing a bug since we weren't closing the widget window in case of initial failure (e.g. insecure context) which would make the widget disappear but the window would still be active (fully transparent) and stuck there until app reload.